document available host preflights types (#656)
replicatedhq/kurl.sh: src/markdown-pages/create-installer/host-preflights/memory.md

---
path: "/docs/create-installer/host-preflights/memory"
date: "2022-01-13"
weight: 26
linktitle: "Memory"
title: "Memory"
---
 

> The memory host preflight check can be used to detect and validate the total amount of memory on the machine.

Reviewer: ```suggestion
The memory host preflight check is used to detect and validate the total amount of memory on the machine.
```
